target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -345,6 +345,11 @@ tplink,tl-wr902ac-v1)
 	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth0"
 	ucidef_set_led_netdev "internet" "Internet" "green:internet" "eth0"
 	;;
+tplink,tl-wr941hp-v1)
+	ucidef_set_led_netdev "wan_data" "WAN Data" "blue:wan_data" "eth1"    "tx rx"
+	ucidef_set_led_netdev "wan_link" "WAN Link" "red:wan_link"  "eth1"    "link"
+	ucidef_set_led_switch "lan"      "LAN"      "blue:lan"      "switch0" "0x1e"
```

Comment:
Please format this like all the other do.

Reply:
And please move this due to alphabetic sorting.

Reply:
Sorry. Done.